robot/basestation/static/js/elements/feed/camera.js

```diff
@@ -286,7 +305,8 @@ $(document).ready(() => {
 
     let streamURL = getStreamURL(getCameraFilename(cameraPanel) + TOPIC_SUFFIX)
     let isPoweredOn = cameraPower.attr("power-on") == "true"
```

Comment:
for future reference, since this is definitely beyond the scope of this PR and should really have it's own issue relating to cleaning up messy design from before, I will just mention that there are a lot of occurrences of this bad practice of relying on GUI button attributes to determine the state of real things on the rover.

I would propose that instead we should use more and more topics (though I know there may be some sort of limit, so keeping that in mind) that would essentially publish states like these from the rover side. So then our buttons would subscribe to those topics and react accordingly.